### PR TITLE
Adding missing info in Order Status doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request (PR) describes adding missing info in Order Status doc

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Order Status

## Affected documentation pages

https://docs.magento.com/user-guide/sales/order-status.html

## Links to Magento source code or PRs

- http://www.magedevelopers.com/blog/magento-tutorials/order-status-magento/#:~:text=%2D%20Pending%20Payment%20%2D%20This%20is%20the,will%20change%20when%20customer%20pays
- https://www.magemoto.com/blog/post/order-status

### CODE REVIEW

**Comment by leslietilling** [comment]
@dnahirna  This one has same issue as #905 did with tax settings topic commit included by mistake and causing conflict.

**Comment by dnahirna** [comment]
> @dnahirna This one has same issue as #905 did with tax settings topic commit included by mistake and causing conflict.

Done!

File: src/sales/order-status.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
-|--- |--- |
-|Processing|processing|
-|Suspected Fraud|fraud|
-|Pending Payment|pending_payment|
-|Payment Review|payment_review|
-|Pending|pending|
-|On Hold|holded|
+|Order Status|Status Code||
+|--- |--- |--- |
+|Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|
```
Comment (leslietilling) [inline src/sales/order-status.md:16]: We are avoiding the use of quotes for literal values.

```suggestion
|Processing|processing|When the state of new orders is set to `Processing`, the _Automatically Invoice All Items_ option becomes available in the configuration.|
```

File: src/sales/order-status.md [hunk 2]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```
Comment (leslietilling) [inline src/sales/order-status.md:17]: Same as previous PR, eway is removed from 2.4

```suggestion
|Suspected Fraud|fraud|Sometimes orders paid via PayPal or another payment gateway are marked as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

File: src/sales/order-status.md [hunk 3]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that customer has been sent to payment gateway website but no return information has been received yet. This status will change when customer pays.|
```
Comment (leslietilling) [inline src/sales/order-status.md:18]:
```suggestion
|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that the customer was directed to the payment gateway website, but no return information has been received yet. This status will change when customer pays.|
```

File: src/sales/order-status.md [hunk 4]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|Pending|pending|This status means no invoice and shipments have been submitted.|
+|On Hold|holded|This status can only be assigned manually. You can put any order on hold.|
 |Open|STATE_OPEN|
-|Complete|complete|
-|Closed|closed|
-|Canceled|canceled|
-|PayPal Canceled Reversal|paypay_canceled_reversal|
-|Pending PayPal|pending_paypal|
-|PayPal Reversed|paypal_reversed|
+|Complete|complete|This status means that order is crated, paid and shipped to customer.|
```
Comment (leslietilling) [inline src/sales/order-status.md:23]:
```suggestion
|Complete|complete|This status means that the order is created, paid, and shipped to customer.|
```

File: src/sales/order-status.md [hunk 5]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|Closed|closed|This status indicates that an order have had a credit memo assigned to it and the customer has received a refund.|
```
Comment (leslietilling) [inline src/sales/order-status.md:24]:
```suggestion
|Closed|closed|This status indicates that an order was assigned a credit memo and the customer has received a refund.|
```

File: src/sales/order-status.md [hunk 6]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|Canceled|canceled|This status is assigned manually by you or, on some payment gateways, when customer does not pay within the specified time.|
```
Comment (leslietilling) [inline src/sales/order-status.md:25]:
```suggestion
|Canceled|canceled|This status is assigned manually in the Admin or, for some payment gateways, when the customer does not pay within the specified time.|
```

File: src/sales/order-status.md [hunk 7]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|PayPal Canceled Reversal|paypay_canceled_reversal|This status means that Paypal cancelled the reversal.|
```
Comment (leslietilling) [inline src/sales/order-status.md:26]:
```suggestion
|PayPal Canceled Reversal|paypay_canceled_reversal|This status means that PayPal canceled the reversal.|
```

File: src/sales/order-status.md [hunk 8]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|Pending PayPal|pending_paypal|This status means that order has been received by PayPal, but payment has not yet been processed.|
```
Comment (leslietilling) [inline src/sales/order-status.md:27]:
```suggestion
|Pending PayPal|pending_paypal|This status means that the order was received by PayPal, but payment has not yet been processed.|
```

File: src/sales/order-status.md [hunk 9]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
+|PayPal Reversed|paypal_reversed|Reversal means that PayPal reversed the transaction.|
```
Comment (leslietilling) [inline src/sales/order-status.md:28]:
```suggestion
|PayPal Reversed|paypal_reversed|This status means that PayPal reversed the transaction.|
```

File: src/sales/order-status.md [hunk 10]
```diff
@@ -11,18 +11,18 @@ _Order Status_
...
-|Pending Payment|pending_payment|
-|Payment Review|payment_review|
-|Pending|pending|
-|On Hold|holded|
+|Order Status|Status Code||
+|--- |--- |--- |
+|Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|
+|Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
+|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that customer has been sent to payment gateway website but no return information has been received yet. This status will change when customer pays.|
+|Payment Review|payment_review|This status appear when PayPal payment review is turned on.|
```
Comment (leslietilling) [inline src/sales/order-status.md:19]:
```suggestion
|Payment Review|payment_review|This status appears when PayPal payment review is turned on.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Most grammatical/editorial changes. Thanks for the contribution!

File: src/sales/order-status.md (hunk 1 above)
Comment (dnahirna) [reply src/sales/order-status.md:16]: Done!

File: src/sales/order-status.md (hunk 2 above)
Comment (dnahirna) [reply src/sales/order-status.md:17]: Done!

File: src/sales/order-status.md (hunk 3 above)
Comment (dnahirna) [reply src/sales/order-status.md:18]: Done!

File: src/sales/order-status.md (hunk 4 above)
Comment (dnahirna) [reply src/sales/order-status.md:23]: Done!

File: src/sales/order-status.md (hunk 5 above)
Comment (dnahirna) [reply src/sales/order-status.md:24]: Done!

File: src/sales/order-status.md (hunk 6 above)
Comment (dnahirna) [reply src/sales/order-status.md:25]: Done!

File: src/sales/order-status.md (hunk 7 above)
Comment (dnahirna) [reply src/sales/order-status.md:26]: Done!

File: src/sales/order-status.md (hunk 8 above)
Comment (dnahirna) [reply src/sales/order-status.md:27]: Done!

File: src/sales/order-status.md (hunk 9 above)
Comment (dnahirna) [reply src/sales/order-status.md:28]: Done!

File: src/sales/order-status.md (hunk 10 above)
Comment (dnahirna) [reply src/sales/order-status.md:19]: Done!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**